code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -304,6 +343,92 @@
 		busy = 0
 		stop_automated_movement = 0
 
+/mob/living/simple_animal/hostile/giant_spider/verb/web()
+	set name = "Spin Web"
+	set desc = "Create a web that slows down movement."
+	set category = "Greimorian"
+
+	var/obj/effect/spider/stickyweb/W = locate() in get_turf(src)
+	if(!W)
+		to_chat(usr, SPAN_NOTICE("\The [src] begins to secrete a sticky substance."))
+		if(!do_after(src, 20)) return
```

Comment:
- Two lines, move the return on its own line
- This allows the gremorian to make a web every 2 seconds, that's a lot of webs, you sure you want it balanced at 2 seconds interval?

Reply:
I want to see how things play out before I make any further changes to timing.